src/views/blocks/tickets.php

```diff
@@ -28,9 +31,15 @@
 if ( ! $provider || empty( $tickets ) ) {
 	return false;
 }
-?>
 
-<?php $this->template( 'blocks/attendees/order-links' ); ?>
+$html = $this->template( 'blocks/attendees/order-links', [], false );
+
+if ( empty( $html) ) {
+	$html = $this->template( 'blocks/attendees/view-link', [], false );;
+}
+
+echo $html;
```

Comment:
:warning: Expected next thing to be an escaping function (see Codex for 'Data Validation'), not '$html'

<sub>hash: f0106463648aa6afb190269daab27b1c</sub>